common/util_test.go

```diff
@@ -422,3 +423,14 @@ func TestAwaitWaitGroup(t *testing.T) {
 		close(doneC)
 	})
 }
+
+func TestValidIDLength(t *testing.T) {
+	t.Run("valid id length, no warnings", func(t *testing.T) {
+		got := ValidIDLength("12345", nil, 7, 10, 0, "", nil, tag.Tag{})
+		require.True(t, got)
+	})
+	t.Run("non valid id length", func(t *testing.T) {
+		got := ValidIDLength("12345", nil, 1, 4, 0, "", nil, tag.Tag{})
```

Comment:
It's better to have a comment (inline is fine as well) on why it is not valid? Because when I look at the ID itself, they are identical in both cases.

Reply:
Fixed it 👍